slack-moji/models/user.rb

```diff
@@ -1,22 +1,22 @@
 class User
   include Mongoid::Document
   include Mongoid::Timestamps
+  include Mongoid::Enum
 
   field :user_id, type: String
   field :user_name, type: String
   field :access_token, type: String
 
-  field :emoji, type: Boolean, default: false
-  field :emoji_count, type: Integer, default: 0
   field :is_bot, type: Boolean, default: false
+  enum :status, %i[unsubscribed emoji] # , :github]
```

Comment:
`enum :status, [:unsubscribed, :emoji]` feels like what I'm actually doing IMO, but I'm not gonna fight rubocop

Reply:
```
rubocop -a ; rubocop --auto-gen-config
```